src/polar-chart/polar-chart.component.scss

```diff
@@ -0,0 +1,24 @@
+.polar-chart {
+  .polar-chart-background {
+    fill: rgb(30, 34, 46);
```

Comment:
This one needs to be the default color for the light theme. The dark colors should be defined as overrides in the demo.

![image](https://cloud.githubusercontent.com/assets/8687/25874457/f0e8da48-3512-11e7-89b5-1c8df2a7cf96.png)